### ISSUE TITLE
enwiki redirect From_I"s

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#444]
following on from #443 
the page en.wikipedia.org/wiki/From_I"s redirect (incorrectly) to a 'Page not found'
The double quote is causing the issue

First, the redirect link on the page shows as en.wikipedia.org/wiki/From_I
The double quote is terminating this

typing in en.wikipedia.org/wiki/From_I"s?redirect=no
give a line

    REDIRECT I"s
Hovering over `I"s` gives `en.wikipedia.org/wiki/I%26quot;s`
clicking on that link gives 'Page not found'

for urls the double quote should be %22 and I think for title should be \&quot;

**Comment by desb42** [comment]
another example of double quote problems
en.wikipedia.org/wiki/Mr._Yankovic

    REDIRECT "Weird Al" Yankovic

**Comment by gnosygnu** [comment]
Yeah, similar root cause as #443. The redirect_mgr is not url decoding the &amp;quot; properly. Instead it's converted the &amp; to %26 and leaving the rest as is.

Will be fixed with #443. Other details for my reference below

---

* Url: en.wikipedia.org/wiki/From_I"s?redirect=no&action=edit
* Source:
```
#REDIRECT [[I&quot;s]] {{R with possibilities}}
```
